src/pip-delete-this-directory.txt

```diff
@@ -0,0 +1,5 @@
+This file is placed here by pip to indicate the source was put
```

Comment:
Please, remove this file. It should have been in your .gitignore to prevent you committing it.